Makefile

```diff
@@ -108,6 +108,37 @@ format: $(GOPATH)/bin/goimports
 	$Q find . -iname \*.go | grep -v \
         -e "^$$" $(addprefix -e ,$(IGNORED_PACKAGES)) | xargs goimports -w
 
+# skaffold.run deploys the virtual-kubelet to the Kubernetes cluster targeted by the current kubeconfig using skaffold.
+.PHONY: skaffold.run
+skaffold.run: MODE ?= dev
```

Comment:
Maybe a `skaffold.clean` as well?

Reply:
I agree we need a goal for cleaning up. `skaffold` already provides a `MODE=delete` so let's use that. I'm renaming the current goal from `skaffold.run` to `skaffold` and changing docs to document all `MODE` and `PROFILE`.

Reply:
`make skaffold MODE=delete` is now documented.